docker/docker-entrypoint.sh

```diff
@@ -3,37 +3,41 @@
 set -e
 
 # Get MuJoCo key from the environment
-echo "${MJKEY}" > /root/.mujoco/mjkey.txt
+echo -e "${MJKEY}" > /root/.mujoco/mjkey.txt
```

Comment:
why -e??

Reply:
Since I'm passing the MJKEY as a build argument in docker, I have to pass it as a single line string, that is, replace the new lines and returns as \n and \r, otherwise the make recipes are ruined by expanding these characters.
By using "echo -e", the backslash characters are expanded to reconstruct mjkey.txt.